### PR TITLE
[Language] Rename DocumentType to HealthcareDocumentType

### PR DESCRIPTION
**Pull request by mssfang**
DocumentType is only for FHIR which is only healthcare feature-related. Is possible to rename to add the prefix "healthcare" to the type name

### CODE REVIEW

File: dev/cognitiveservices/data-plane/Language/analyzetext.json [hunk 1]
```diff
@@ -783,9 +783,9 @@
             "modelAsString": true
           }
         },
-        "documentType": {
+        "healthcareDocumentType": {
```
Comment (deyaaeldeen) [inline dev/cognitiveservices/data-plane/Language/analyzetext.json:786]: I think we would like to keep the property name as `documentType` and update its type only.

Comment (joseharriaga) [inline dev/cognitiveservices/data-plane/Language/analyzetext.json:786]: The name of the property should probably be kept the same, and the name of the enum is that only thing that changes.

Comment (mssfang) [inline dev/cognitiveservices/data-plane/Language/analyzetext.json:786]:
```suggestion
        "documentType": {
```

**Review by heaths — APPROVED** [review]

**Comment by heaths** [comment]
The lintdiff error may have been resolved. Please rebase your branch on the latest in main or, @kayousef, was that only fixed in the 2022-10-01-preview branch and hasn't been merged yet?

@mssfang I can't merge a PR with failed checks, but this was fixed.

**Comment by mssfang** [comment]
> The lintdiff error may have been resolved. Please rebase your branch on the latest in main or, @kayousef, was that only fixed in the 2022-10-01-preview branch and hasn't been merged yet?
> 
> @mssfang I can't merge a PR with failed checks, but this was fixed.

Thanks @heaths. This branch should be based on the latest SHA from main branch when I created it earlier today.

**Review by aurghob — APPROVED** [review]

**Comment by lmazuel** [comment]
Talked with @mssfang offline to get this merged

**Pull request merged**